src/main/java/org/railwaystations/api/model/Station.java

```diff
@@ -136,6 +139,14 @@ public String getLicense() {
         return license;
     }
 
+    public String getLicenseUrl() {
+        return licenseUrl;
+    }
+
+    public void setLicenseUrl(String licenseUrl) {
```

Comment:
The Station class was immutable, now we introduce a mutator method here.

Reply:
Also the Station instances are cached and used to serve several requests. I think we'll get some race conditions when we two requests are processed with different languages.